hack/scripts/generate-configmaps.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
+# Generate Prometheus configuration ConfigMap
+kubectl create configmap --dry-run=true prometheus-k8s --from-file=assets/prometheus/ -oyaml > manifests/prometheus/prometheus-k8s-cm.yaml
```

Comment:
Do you think there might be new files in this directory? If not, then just for the sake of understanding at first, sight we can select the config file directly in the `--from-file` flag instead of the entire directory. Feel free to ignore if you disagree.